source/contracts/trading/FillOrder.sol

```diff
@@ -368,7 +368,7 @@ contract FillOrder is CashAutoConverter, ReentrancyGuard, IFillOrder {
         uint256 _creatorCashBalance = _tradeData.contracts.denominationToken.balanceOf(_tradeData.creator.participantAddress);
         if (_creatorCashBalance > 0) {
             _tradeData.contracts.augur.trustedTransfer(_tradeData.contracts.denominationToken, _tradeData.creator.participantAddress, this, _creatorCashBalance);
-            _tradeData.contracts.denominationToken.withdrawEtherTo(_tradeData.creator.participantAddress, _creatorCashBalance);
+            _tradeData.contracts.denominationToken.withdrawEtherToIfPossible(_tradeData.creator.participantAddress, _creatorCashBalance);
```

Comment:
Never noticed this here before, so denomination HAS to be cash, for this version